auth/space_collaborators_client.go

```diff
@@ -13,7 +13,14 @@ import (
 	"github.com/goadesign/goa/uuid"
 )
 
-func GetSpaceCollaborators(ctx context.Context, client *api.Client, spaceID uuid.UUID) (*api.UserList, error) {
+type CollaboratorCollector interface {
```

Comment:
Was this change made for better coding practices? ( 'programming to an interface' ) 

Btw, would/should similar practice be applied to the methods inside `auth/user_client.go` ?

Reply:
I needed an interface for the tests to be able to mock the Auth service.

Reply:
Ah ok, right.